openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -509,6 +509,64 @@ def _default_settings(cls):
             output_settings=MultiStateOutputSettings(),
         )
 
+    @classmethod
+    def _adaptive_settings(
+            cls,
+            stateA: ChemicalSystem,
+            stateB: ChemicalSystem,
+            mapping: gufe.LigandAtomMapping | list[gufe.LigandAtomMapping]
+    ) -> RelativeHybridTopologyProtocolSettings:
+        """
+        Get the recommended OpenFE settings for this protocol based on the input states involved in the
+        transformation.
+
+        These are intended as a suitable starting point for creating an instance of this protocol, which can be further
+        customized before performing a Protocol.
+
```

Comment:
Something something let's add parameters and returns docstring inputs (once this is out of draft)